app.py

```diff
@@ -29,6 +29,7 @@
     'en': '🇺🇸',
     'nl': '🇳🇱',
     'es': '🇪🇸',
+    'fr': 'ғʀ',
```

Comment:
Can you give me a link, or a link on how to type that symbol?

Reply:
I typed it, and to tell the truth, it was the most difficult part of the translation.
I used an online font converter (probably this one if I remember correctly https://manytools.org/facebook-twitter/smallcaps-text-tool/ ) to get the text to small caps. 

I’m not sure it’s exactly the intended formatting @Felienne expected, but when I run Hedy, the dropdown menu with the languages looks fine with this, so I guess it’s correct.

Reply:
Yeah it all works fine, that a lot, it just misses the flag, but I can add that myself!